dev-haskell/quickcheck-instances/quickcheck-instances-0.3.32-r1.ebuild

```diff
@@ -24,7 +24,7 @@ RDEPEND=">=dev-haskell/case-insensitive-1.2.0.11:=[profile?] <dev-haskell/case-i
 	>=dev-haskell/integer-logarithms-1.0.3.1:=[profile?] <dev-haskell/integer-logarithms-1.1:=[profile?]
 	>=dev-haskell/old-time-1.1.0.0:=[profile?] <dev-haskell/old-time-1.2:=[profile?]
 	>=dev-haskell/onetuple-0.4.2:=[profile?] <dev-haskell/onetuple-0.5:=[profile?]
-	>=dev-haskell/primitive-0.9.0.0:=[profile?] <dev-haskell/primitive-0.10:=[profile?]
+	>=dev-haskell/primitive-0.9.0.0:=[profile?]
```

Comment:
Another removal of an upper bound with no change to `CABAL_HACKAGE_REVISION` or `CABAL_CHDEPS`.